server/src/lib/routes/verify/get_basic_auth.ts

```diff
@@ -52,7 +52,7 @@ export default function (req: Express.Request, res: Express.Response,
       return vars.usersDatabase.checkUserPassword(username, password);
     })
     .then(function (groupsAndEmails) {
-      return AccessControl(req, vars, domain, originalUri, username, groupsAndEmails.groups)
+      return AccessControl(req, vars, domain, originalUri, username, groupsAndEmails.groups, WhitelistValue.NOT_WHITELISTED)
```

Comment:
Isn't the idea of utilising Basic Auth that it would occur before it hits Authelia with the auth_request module?
I can't tell because I can't find any examples where you have utilised basic auth. If the above use case is correct, the basic auth prompt will always appear for a user and we would not have any means to prevent this prompt from appearing.

Having said that we could treat the user based on a whitelisted address post that fact. For example they could just press enter without any details being entered (or enter in completely incorrect details) but I feel like this defeats the purpose of bypassing auth, which is kinda why we decided not in include whitelisting for basic auth.

Happy to adjust if our assumptions are incorrect but would probably need some guidance on how exactly it's intended to work with an example :).